libobs/obs-scene.c

```diff
@@ -608,24 +608,14 @@ static void render_item_texture(struct obs_scene_item *item,
 
 	float multiplier = 1.f;
 
-	switch (current_space) {
-	case GS_CS_709_SCRGB:
-		switch (source_space) {
-		case GS_CS_SRGB:
-		case GS_CS_SRGB_16F:
-		case GS_CS_709_EXTENDED:
-			multiplier = obs_get_video_sdr_white_level() / 80.f;
-		}
+	if (current_space == GS_CS_709_SCRGB &&
+	    source_space != GS_CS_709_SCRGB) {
+		multiplier = obs_get_video_sdr_white_level() / 80.f;
```

Comment:
I see we're effectively switching from a whitelist to a blacklist. Could this produce unintended behavior if more color spaces are added in the future?

Reply:
Is it not possible that another color space with its own multiplier could be added, that would require fraction multiplication instead? Say, a hypothetical `GS_CS_420_BLZIT` where it requires a denominator of 75.0f?

Reply:
I'm just worried that in a future where this happens, we are placing a bet on a future developer being able to catch this behavior, and going from a switch here to a single if statement would likely make troubleshooting errors more difficult.

A comment perhaps pointing out that this logic relies on there only being one color space with a denominator would be good IMO.